config/gpu_configs/gpu_aquaplanet_dyamond_ss_1process.yml

```
@@ -0,0 +1,20 @@
+dt_save_state_to_disk: "Inf"
+dt_save_to_sol: "Inf"
+output_default_diagnostics: false
+h_elem: 30
+z_max: 55000.0
+z_elem: 63
+dz_bottom: 30.0
+dz_top: 3000.0
+moist: "equil" 
+precip_model: "0M" 
+rad: "allskywithclear" 
+idealized_insolation: false
+dt_rad: "1hours"
+vert_diff: "true" 
```

```suggestion
vert_diff: "FriersonDiffusion"
implicit_diffusion: true
approximate_linear_solve_iters: 2
```

following @szy21 Slack comment. cc @nefrathenrici

---

Thanks! Also we should set `dt_cloud_fraction: "1hours"`. It would be easier to just rename the `gpu_aquaplanet_dyamond.yml` file.